scripts/eject-typescript.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env zx
```

Comment:
I'll remove this script I think. I rarely use eject scripts and in the off chance I do they tend to never quite work. 😊